frontend/src/entities/towers/Tower.js

```diff
@@ -5,4 +5,9 @@ export default class Tower extends Entity {
         super(x, y, z, scale);
         this.hp = Infinity;
     }
+
+    rotateTowardsTarget(target) {
+        const [dx, dy] = [target[0]-this.x, target[1]-this.y];
+        this.rotation[1] = Math.atan2(-dy, dx); // around render coordinates z axis
```

Comment:
Ah, doesn't look like we're converting Euler rotation when we render it. I'll cover that under #30 as well.